### PR TITLE
Remove nodes and edges

### PR DESCRIPTION
**Pull request by meganle**
It's time @AAAstorga @matthewcoufal 

- Removed nodes and edges
- Removed V1 routes
- Alphabetized lots of imports lol
- Make type checking for return values for routers strong
- Refactored tests to reflect the change and addressed #140

### CODE REVIEW

File: src/API.js [hunk 1]
```diff
@@ -25,24 +25,10 @@ class API {
...
-              .map(r => `/api/v1${r.route.path}`));
-
-          this.express.use('/api/v1', router);
-      });
-
       // Connect all routers in ./routers/v2
       const opts2 = { cwd: path.join(__dirname, 'routers/v2') };
       globby.sync(['**/*Router.js'], opts2).forEach((file) => {
-      // All v2 routes
+          /* eslint-disable global-require */
```
Comment (young-k) [inline src/API.js]: indentation looks off here?

File: src/utils/Lib.js [hunk 2]
```diff
@@ -44,7 +44,8 @@ async function ensureAuthenticated(req: Request, res: Response,
     }
 
     if (!await UserSessionsRepo.verifySession(bearerToken)) {
-        res.send(401, 'Invalid session token');
+        res.status(401).send(new AppDevResponse(false,
```
Comment (young-k) [inline src/utils/Lib.js]: isn't it more idiomatic to have `false` on a separate line here?

File: test/routes/api.draft.test.js [hunk 3]
```diff
@@ -29,20 +29,20 @@ test('Create a draft', async () => {
         text: 'Test draft',
         options: ['yes', 'no'],
     };
-    const getstr = await request(post('/drafts/', body, userToken));
-    const getres = getstr;
-    draft1 = getres.data.node;
-    expect(getres.success).toBe(true);
-    expect(draft1).toMatchObject(body);
+    await request(post('/drafts/', body, userToken)).then((getres) => {
```
Comment (kevinchan159) [inline test/routes/api.draft.test.js]: is `getRes` better?

File: test/routes/api.poll.test.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
+import request from 'request-promise-native';
 import dbConnection from '../../src/db/DbConnection';
 import UsersRepo from '../../src/repos/UsersRepo';
 import UserSessionsRepo from '../../src/repos/UserSessionsRepo';
 import GroupsRepo from '../../src/repos/GroupsRepo';
```
Comment (kevinchan159) [inline test/routes/api.poll.test.js]: unrelated to the PR but G < U

**Pull request merged**